client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js

```diff
@@ -64,8 +64,9 @@ const DataFilesProjectEditDescriptionModal = () => {
   const validationSchema = Yup.object().shape({
     title: Yup.string()
       .min(1)
+      .max(70)
       .required('Please enter a title.'),
-    description: Yup.string()
+    description: Yup.string().max(800)
   });
```

Comment:
__Comment__

- This validation uses `Yup`.
- The validation for the search field uses HTML5 ([initial implementation](https://github.com/TACC/Core-Portal/pull/408/files), [planned follow-up updates](https://jira.tacc.utexas.edu/browse/FP-1096)).

HTML5 is based on web standards, has its own API, and is great for [simple input validation](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Constraint_validation).

This is not a request for change, but an example of a difference in implementation for the Portal team to consider. @rstijerina's feedback may be helpful determining whether and how to worry about this.

Reply:
@rstijerina @tacc-wbomar  Should I add `min`/`max` as a prop in `FormField` to be used in the `Input` component? 

Relative [documentation](https://reactstrap.github.io/components/input-group/).

Reply:
@tacc-wbomar Does design have a strong preference? Formik/Yup is strongly preferred for complex validation, but for simple validation like this, true it could be HTML5 validation. Should we decide on something for consistency?

@duckonomy If both HTML validation and yup validation is present, will both show? For now, I'd say no need to add min/max as a prop in FormField unless we decide we're going with HTML5 validation on Input Fields

Reply:
Thanks, Sal. Great points. Let's hold off for now. Design does not have a preference yet.

<details>
<summary>Details</summary>

I have not worked with Design to define validation UI. The two implementation options probably appear differently right now. Design should standardize UI to assist any possible change in implementation.

</details>